windows/installer-source/KaliteSetupScript.iss

```diff
@@ -337,6 +337,25 @@ begin
     end;
 end;
 
+procedure HandlePipSetup;
+var
+    PipCommand: string;
+    PipPath: string;
+    ErrorCode: integer;
+
+begin
+    PipPath := 'C:\Python27\Scripts\pip.exe';
+    PipCommand := 'install ' + '"' + ExpandConstant('{app}') + '\ka-lite\dist\ka-lite-static-0.14.dev10.zip' + '"';

```

Comment:
Let's make it `ka-lite-static.zip` so that the script to generate the zip file can use that on the build server?  Unless it's necessary to put the version in the filename?